zone/spell_effects.cpp

```diff
@@ -2950,6 +2950,20 @@ bool Mob::SpellEffect(Mob* caster, uint16 spell_id, float partial, int level_ove
 				break;
 			}
 
+			case SE_PetShield: {
+				if (IsPet()) {
+					Mob* petowner = GetOwner();
+					if (petowner) {
+
+						int shield_duration = spells[spell_id].base[i] * 12 * 1000;
+						int m_shield_target_mitigation = spells[spell_id].base2[i] ? spells[spell_id].base2[i] : 50;
```

Comment:
This shouldn't prefix with `m_` as this is not a member variable of a class

Reply:
![image](https://user-images.githubusercontent.com/3319450/129508840-a99e9613-ac82-4217-9bae-fbd136e95723.png)